tests/integration/feeds/test_feeds_client.py

```diff
@@ -0,0 +1,170 @@
+PACKAGE_PATH = (
+    Path(__file__).parent.resolve()
+    / "test_files"
+    / "sample-measurement_0.5.0_windows_x64.nipkg"
+)
+INVALID_PACKAGE_PATH = Path(__file__).parent.resolve()
+
+
+@pytest.fixture(scope="class")
+def client(enterprise_config) -> FeedsClient:
+    """Fixture to create a FeedsClient instance."""
+    return FeedsClient(enterprise_config)
+
+
+@pytest.fixture(scope="class")
+def create_feed(client: FeedsClient):
+    """Fixture to return a object that creates feed."""
+
+    def _create_feed(feed):
+        response = client.create_feed(feed)
+        return response
+
+    yield _create_feed
+
+
+@pytest.fixture(scope="class")
+def create_windows_feed_request():
+    """Fixture to create a request body of create feed API for windows platform."""
+
+    def _create_feed_request():
+        feed_request = CreateFeedRequest(
+            name=WINDOWS_FEED_NAME,
+            platform=Platform.WINDOWS.value,
+            description=FEED_DESCRIPTION,
+        )
+        return feed_request
+
+    yield _create_feed_request
+
+
+@pytest.fixture(scope="class")
+def create_linux_feed_request():
+    """Fixture to create a request body of create feed API for linux platform."""
+
+    def _create_feed_request():
+        feed_request = CreateFeedRequest(
+            name=LINUX_FEED_NAME,
+            platform=Platform.NI_LINUX_RT.value,
+            description=FEED_DESCRIPTION,
+        )
+        return feed_request
+
+    yield _create_feed_request
+
+
+@pytest.fixture(scope="class")
+def get_windows_feed_id(client: FeedsClient):
+    """Fixture to return the Feed ID of the created windows feed."""
+    query_feeds_response = client.query_feeds(platform=Platform.WINDOWS.value)
+
+    for feed in query_feeds_response.feeds:
+        if feed.name == WINDOWS_FEED_NAME:
+            return feed.id
+
+
+@pytest.fixture(scope="class")
+def get_linux_feed_id(client: FeedsClient):
+    """Fixture to return the Feed ID of the created Linux feed."""
+    query_feeds_response = client.query_feeds(platform=Platform.NI_LINUX_RT.value)
+
+    for feed in query_feeds_response.feeds:
+        if feed.name == LINUX_FEED_NAME:
+            return feed.id
+
+
+@pytest.mark.enterprise
+@pytest.mark.integration
+class TestFeedsClient:
+    """Class contains a set of test methods to test SystemLink Feeds API."""
+
+    def test__create_feed_windows_platform(
+        self,
+        create_feed: Callable,
+        create_windows_feed_request: Callable,
+    ):
+        """Test the case of a completely successful create feed API for windows platform."""
+        request_body = create_windows_feed_request()
+        response = create_feed(request_body)
+
+        assert response.id is not None
+        assert response.workspace is not None
+        assert response.name == WINDOWS_FEED_NAME
+        assert response.platform == Platform.WINDOWS
+        assert response.description == FEED_DESCRIPTION
+
+    def test__create_feed_linux_platform(
```

Comment:
The test names are inconsistent with the other tests in the repo. It should be of the format `test__<when>__<then>` where "when" corresponds to the action performed and "then" highlights the response/outcome.